projects/igniteui-angular/src/lib/data-operations/filtering-expression.interface.ts

```diff
@@ -14,9 +14,9 @@ export enum FilteringLogic {
  */
 export declare interface IFilteringExpression {
     fieldName: string;
-    condition?: IFilteringOperation;
-    conditionName?: string;
-    searchVal?: any;
-    searchTree?: IExpressionTree;
+    condition?: IFilteringOperation | null;
+    conditionName?: string | null;
+    searchVal?: any | null;
```

Comment:
any is already null? :) Kinda redundant on this one.

Reply:
Fair